docs/snippets/web-components/simple-page-implementation.js.mdx

```diff
@@ -0,0 +1,28 @@
+```js
```

Comment:
@kasperpeulen — Does this implementation look right to you?

Used here: https://storybook.js.org/docs/7.0/web-components/writing-stories/build-pages-with-storybook#args-composition-for-presentational-screens

Reply:
looks good as well